creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -631,51 +892,37 @@ class SnapbackSM {
         time: Date.now()
       })
 
-      // Build map of secondary node to secondary user wallets array
-      const secondaryNodesToUserWalletsMap = this.buildSecondaryNodesToUserWalletsMap(potentialSyncRequests)
-      decisionTree.push({
-        stage: 'buildSecondaryNodesToUserWalletsMap() Success',
-        vals: { numSecondaryNodes: Object.keys(secondaryNodesToUserWalletsMap).length },
-        time: Date.now()
-      })
-
-      // Retrieve clock statuses for all secondary users from secondary nodes
-      let secondaryNodesToUserClockStatusesMap
+      // Setup the mapping of Content Node endpoint to service provider id. Used in reconfig
       try {
-        secondaryNodesToUserClockStatusesMap = await this.retrieveClockStatusesForSecondaryUsersFromNodes(
-          secondaryNodesToUserWalletsMap
-        )
+        await this.peerSetManager.updateEndpointToSpIdMap(this.audiusLibs.ethContracts)
+        // Reset `this.highestReconfigModeEnabled` to the enabled mode
+        this.highestReconfigModeEnabled = this.getHighestReconfigModeEnabled()
+
         decisionTree.push({
-          stage: 'retrieveClockStatusesForSecondaryUsersFromNodes() Success',
-          vals: { },
+          stage: `updateEndpointToSpIdMap() Success`,
+          vals: {
+            endpointToSPIdMapSize: Object.keys(this.peerSetManager.endpointToSPIdMap).length
+          },
           time: Date.now()
         })
       } catch (e) {
+        // If unable to initialize `endpointToSpIdMap`, disable reconfig
+        this.highestReconfigModeEnabled = RECONFIG_MODES.RECONFIG_DISABLED.key
```

Comment:
we just need to recalculate the `this.enabledReconfigModes` member variable when `this.highestReconfigModeEnabled` gets updated. 

I think this is the best way to approach the super setting logic with these reconfig modes. do you have another proposal?

Reply:
wanted to find a way to avoid managing these two connected variables independently, opened a PR here - lmk wyt!
https://github.com/AudiusProject/audius-protocol/pull/1645

Reply:
approved!